java/bulk-import/bulk-import-runner/src/test/java/sleeper/bulkimport/job/runner/BulkImportJobDriverIT.java

```diff
@@ -528,27 +523,26 @@ void shouldNotThrowExceptionIfProvidedWithDirectoryWhichContainsParquetAndNonPar
                 .extracting(FileReference::getNumberOfRecords, FileReference::getPartitionId,
                         file -> readRecords(file.getFilename(), schema))
                 .containsExactly(tuple(200L, expectedPartitionId, records));
-        assertThat(statusStore.getAllJobs(tableProperties.getId())).containsExactly(
+        assertThat(statusStore.getAllJobs(tableProperties.get(TABLE_ID))).containsExactly(
                 finishedIngestJobWithValidation(job.toIngestJob(), taskId, validationTime,
                         summary(startTime, endTime, records.size(), records.size())));
     }
 
-    @ParameterizedTest
-    @MethodSource("getParameters")
+    @ParameterizedTest @MethodSource("getParameters")
```

Comment:
This is probably accidental formatting, but i find it a bit easier to read if annotations are broken up by newlines.

Reply:
Thanks. I've reverted this and updated the IDE configuration.